### PR TITLE
Errors

### PR DESCRIPTION
**Pull request by teleyinex**
Fix error when PyBossa server does not support keyset pagination.

### CODE REVIEW

**Comment by teleyinex** [comment]
@alejandrodob can you review it?

**Comment by therealmarv** [comment]
@teleyinex What about the Travis errors?

**Comment by therealmarv** [comment]
:cake: if Travis says :green_heart:

File: enki/exceptions.py [hunk 1]
```diff
@@ -26,6 +26,18 @@
 """
 
 
+def check_errors(data):
```
Comment (alejandrodob) [inline enki/exceptions.py]: I would move this to the ServerTaskRunsLoader class as a method. I believe it is its responsibility to know if a response from the server is ok or not.

**Comment by alejandrodob** [comment]
Other than the comment I've made, it is ok to me

File: enki/exceptions.py (hunk 1 above)
Comment (teleyinex) [inline enki/exceptions.py]: Well, I've make it as generic as possible, so we can extend it for future checks. Therefore, I put it in there.

**Comment by alejandrodob** [comment]
:cake: :heart_eyes:

**Pull request merged**